test/emx.test.ts

```diff
@@ -0,0 +1,163 @@
+import { getEmx, getTablename } from '../src/emx'
+import { CatalogueDatatype, MolgenisDataType } from '../src/model'
+
+describe('getTablename', () => {
+  it('returns non_repeated for non-repeated variable', () => {
+    expect(getTablename(1)).toBe('non_repeated')
+  })
+
+  it('returns yearly_repeated for variable repeated twice', () => {
+    expect(getTablename(2)).toBe('yearly_repeated')
+  })
+
+  it('returns yearly_repeated for variable repeated 17 times', () => {
+    expect(getTablename(17)).toBe('yearly_repeated')
+  })
+
+  it('returns trimester_repeated for variable repeated 43 times', () => {
```

Comment:
This is weekly I think? Trimester === 3 times. Weekly variables maximum reoccurrence is currently unknown. I would change this to trimester and limit it on three in the switch.